otmonitor.vfs/otmonitor.tcl

```diff
@@ -2136,6 +2139,12 @@ register 82	{f8.8}		"Exhaust inlet temperature"
 register 83	{f8.8}		"Exhaust outlet temperature"
 register 84	{u16}		"Exhaust fan speed"
 register 85	{u16}		"Inlet fan speed"
+register 98	{f8.8}		"RF sensor status information" ;# Data-type assumed
+register 99	{flag8 flag8}	"Operating Mode HC1, HC2/ Operating mode DHW"
```

Comment:
That still doesn't show the data in a very useful way. I was more thinking of something along the lines of this patch: [pr-patch.txt](https://github.com/hvxl/otmonitor/files/8079468/pr-patch.txt)